_posts/2021-09-02-gsoc-final-report-orion.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: "GSoC Report: Solvation Analysis"
+---
+
+## Retrospective
```

Comment:
It would be really cool to read at the outset about the _science_ that inspired the work, perhaps even your own work. Do you have an image that summarizes the scientific aspects?

It would also be interesting to have a big picture view of what your goals for the package are — what do you want to accomplish. What's the motivation for reading further and learning about your work?

Reply:
I added a couple paragraphs on this! It describes my thinking on where this project is at and why I did it.

Reply:
awesome